www/includes/easyparliament/templates/html/mp/divisions.php

```diff
@@ -0,0 +1,163 @@
+                                        <h3 class="policy-vote-overall-stance">
+                                            <?= $full_name ?> voted <?= $policy['position']['voted'] ?> <?= $policy['desc'] ?>
+                                        </h3>
+
+                                        <?php if ( DEVSITE ) { ?>
+                                        <p class="policy-vote-agree-disagree">
+                                            <button class="button">I agree with this MP</button>
+                                            <button class="button button--negative">I disagree with this MP</button>
+                                        </p>
+                                        <?php } ?>
+
+                                        <h3 class="policy-votes-list-header"><span id="policy-votes-type">All</span> votes about <?= $policy['desc'] ?>:</h3>
+
+                                        <ul class="vote-descriptions policy-votes">
+                                        <?php foreach ($policy['divisions'] as $division) { ?>
+                                            <li id="<?= $division['division_id'] ?>" class="<?= $division['strong'] ? 'policy-vote--major' : 'policy-vote--minor' ?>">
+                                                <span class="policy-vote__date">On <?= strftime('%e %b %Y', strtotime($division['date'])) ?>:</span>
+                                                <span class="policy-vote__text"><?= $full_name ?><?= $division['text'] ?></span>
+                                                <?php if ( $division['url'] ) { ?>
+                                                    <a class="vote-description__source" href="<?= $division['url'] ?>">Show full debate</a>
+                                                <?php } ?>
+                                            </li>
+
+                                        <?php $displayed_votes = TRUE; ?>
+
+                                        <?php } ?>
+                                        </ul>
+
+                                        <div class="policy-votes-list-footer">
+                                            <p class="policy-votes__byline">Vote information from <a href="http://www.publicwhip.org.uk">PublicWhip</a></p>
+                                            <?php if ( $policy['weak_count'] > 0 ) { ?>
+                                            <p><button class="button secondary-button small js-show-all-votes">Show all votes, including <?= $policy['weak_count'] ?> less important <?= $policy['weak_count'] == 1 ? 'vote' : 'votes' ?></button></p>
+                                            <?php } ?>
+                                        </div>
+
+                                        <script type="text/javascript">
+                                        $(function(){
+                                            $('.policy-vote--minor').addClass('js');
+                                            $('.js-show-all-votes').show();
```

Comment:
TWFY adds a `js` class to `body`, so you could not have these two lines, if in the CSS instead of `.policy-vote--minor.js` you had `.js .policy-vote--minor`, and add `.js .js-show-all-votes { display: block; }`, then it would work directly (yay title change :) ).